docs/setup-and-installing.md

```diff
@@ -86,13 +86,13 @@ Currently, you can try this out using the `Playground`.
 
 #### 2. Run the `Playground` locally using SpatialOS
 
+1. Open Unity and open the Unity project located at`UnityGDK\workers\unity`.
```

Comment:
"This causes Unity to generate a Visual Studio solution, `unity.sln`, within `UnityGDK/workers/unity`."
The visual studio solution isn't the interesting part (and may not be generated on all platforms or for all code editors.)

I'd replace the above with:
"Unity will automatically download several required SpatialOS libraries. Unity may open a browser window prompting you to login to your SpatialOS account. If this happens, please login. This will only happen the first time you open you project, or if the version of the required libraries changes."

Reply:
fixed :)